Makefile

```diff
@@ -42,3 +42,17 @@ test-coverage:
 
 clean:
 	rm -rf $(GIT_DIST_PATH)
+
+fuzz:
```

Comment:
Please add `make fuzz` to be executed on its own GitHub Action workflow on a [schedule](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule).

Reply:
> Please add make fuzz to be executed on its own GitHub Action workflow on a [schedule](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule).


I have a better option. how about using `oss-fuzz`?

Reply:
That's a good shout. Please tag me on the PR to `oss-fuzz`.